.github/workflows/main.yml

```diff
@@ -75,6 +90,12 @@ jobs:
       VERSION: ${{ needs.publish.outputs.version }}
       DEVELOPMENT_BRANCH: ${{ github.ref_name }}
     steps:
+      - uses: Activiti/activiti-scripts/.github/actions/wait-for-chart@master
```

Comment:
I don't really follow what we're waiting for: for the helm chart that we just published to be available for use?

Reply:
The Helm chart is published to gh-pages branch, which triggers Github action publish workflow to update Github site. Without wait, there will be problems when promoting new versions downstream trying to lint, build and package new version

Reply:
so we're waiting, potentially for more than 4 minutes, in a running build, for another CI process to go through?

Reply:
Yes, it will not interfere with another CI build on RC tag push

Reply:
The builds for [github pages](https://github.com/Activiti/activiti-cloud-helm-charts/actions) are taking about 2 minutes to run. I don't think we can do much about this (unless migrating from gh pages to something else: same approach used for [Alfresco/charts](https://github.com/Alfresco/charts)? But this is out of of the scope of this epic).